### PR TITLE
added drive to state 100, bug fix for length 0 treajectories in DriveToWaypoint100

### PR DESCRIPTION
**Pull request by 812405**

### CODE REVIEW

File: comp/swerve100/simgui-ds.json
Comment (truher) [inline comp/swerve100/simgui-ds.json:1]: remove this file from the PR

File: comp/swerve100/simgui-window.json
Comment (truher) [inline comp/swerve100/simgui-window.json:1]: remove this file from the PR

File: comp/swerve100/simgui.json
Comment (truher) [inline comp/swerve100/simgui.json:1]: remove this file from the PR

File: comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java
Comment (truher) [inline comp/swerve100/src/main/java/org/team100/frc2024/RobotContainer.java:1]: are there any substantive changes in this file?

**Comment by 812405** [comment]
merge conflicts resolved

**Comment by 812405** [comment]
its happy so im just gonna merge it

**Pull request merged**